dev.py

```diff
@@ -33,8 +36,12 @@ def test(slow):
 
 
 @cli.command()
-def coverage():
-    execute("pytest --cov=werkit")
+@click.option("--slow/--not-slow", default=True)
```

Comment:
What does this slow/not-slow option do?

Reply:
It skips the integration tests on Lambda, which are marked "slow" using `@pytest.mark.slow`.